extensions/amp-jwplayer/amp-jwplayer.md

```diff
@@ -67,16 +67,46 @@ Example:
 ## Attributes
 
 ##### data-player-id
+Type: String
 
 JW Platform player id. This is an 8-digit alphanumeric sequence that can be found in the [Players](https://dashboard.jwplayer.com/#/players) section in your JW Player Dashboard. (**Required**)
 
+
 ##### data-media-id
+Type: String
 
 The JW Platform media id. This is an 8-digit alphanumeric sequence that can be found in the [Content](https://dashboard.jwplayer.com/#/content) section in your JW Player Dashboard. (**Required if `data-playlist-id` is not defined.**)
 
+
 ##### data-playlist-id
+Type: String
+
+The JW Platform playlist id. This is an 8-digit alphanumeric sequence that can be found in the [Playlists](https://dashboard.jwplayer.com/#/content/playlists) section in your JW Player Dashboard.  If both `data-playlist-id` and `data-media-id` are specified, `data-playlist-id` takes 
+precedence.  (**Required if `data-media-id` is not defined.**)
+
+
+##### data-content-search
+Type: String
+Denotes the type of the playlist. This is a search playlist that takes in a keyword or phrase as the search query and generates a playlist based on that search query. If contextual article matching is desired, use the value `_CONTEXTUAL_` (data-content-contextual must also be true).
+
+
+##### data-content-contextual
+Type: Boolean
```

Comment:
All attributes are strings, please specify that it should be `'true'` or `'false'`.